.editorconfig

```diff
@@ -0,0 +1,20 @@
+# Strider Editor/IDE Settings
+# This file is used to promote consistent source code standards
+# amongst all Strider-CD contributors.
+# More information can be found here: http://editorconfig.org/
+
+# General Settings
+root = true
+
+# Settings for all files
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
It ensures that each file has an extra newline at the bottom.  This assists with utilities that concatenate source files to ensure that they do not place the first list of the 2nd file on the same line as they first, for example:

```
/** a great file1.js **/
module.exports {
  ...
}; /** a great file2.js **/
module.exports = {
  ...
}; 
```

Not likely to apply to Strider, at current, but its not harmful either and is generally a good idea.